index.html

```diff
@@ -1,13 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This a portfolio of a full stack web developer">
+    <meta name="keywords" content="porfotlio, software developer, web developer">
+    <meta name="author" content="Julio Quezada">
     <link rel="stylesheet" href="./assets/css/style.css">
     <title>Document</title>
 </head>
+
 <body>
-    <h1 class="makeHeaderRed">My portfolio</h1>
+    <header>
+        <nav class="header-container">
+            <div class="logo">
+                <a href="#"><img src="./assets/images/logo3.png" alt="logo"></a>
+            </div>
```

Comment:
- [ ] Kindly make the logo look identical to that of the Figma template used in your design. Your logo looks very small in size compared to that of the Figma template. I would advise you to use text instead of an image. That way, you would be able to use the CSS declarations used in the Figma design. 👍 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219815342-162d2a78-aadf-4b7b-a610-94030bd08f50.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/219815470-b26550aa-bf48-4286-ae5e-06d71fd3cfa6.png)